lib/temporalio/worker.rb

```diff
@@ -17,10 +23,11 @@ def initialize(
       activity_executor: nil,
       max_concurrent_activities: 100
     )
-      @running = false
+      @started = false
+      @shutdown = false
       @mutex = Mutex.new
       @runtime = Temporalio::Runtime.instance
-      activity_executor ||= ThreadPoolExecutor.new(max_concurrent_activities)
+      @activity_executor = activity_executor || ThreadPoolExecutor.new(max_concurrent_activities)
```

Comment:
I'd add a TODO here to consider max local activities when those become relevant.

Reply:
I'll make a note of it, good call 👍